dist/generator-vulcanjs/lib/ui-text.js

```diff
@@ -2,6 +2,7 @@ const chalk = require('chalk');
 
 const descriptions = {
   appName: 'The name of your app',
+  doShallowClone: 'Default is fast and great for testing. Full vulcan git clone will avoid requiring to unshallow the repository to change the git remote.',
```

Comment:
I'm sure 80% of people wont even know the meaning of `unshallow`. For example I didn't know it, until a week ago.